.travis.yml

```diff
@@ -32,17 +31,44 @@ matrix:
       - HDF5_DIR=$HDF5_CACHE_DIR/$HDF5_VERSION
       - H5PY_ENFORCE_COVERAGE=yes
       os: linux-ppc64le
+    - os: linux
+      services: docker
+      python: 3.7
+      virt: vm
+      group: edge
+      arch: arm64
+      env:
+        - TOXENV=py37-test-deps
+        - HDF5_VERSION=1.10.5
+        - HDF5_DIR=$HDF5_CACHE_DIR/$HDF5_VERSION
+        - H5PY_ENFORCE_COVERAGE=yes
+        - CIBW_BUILD: cp3[789]-*
+        - CIBW_MANYLINUX_AARCH64_IMAGE: ghcr.io/h5py/manylinux2014_aarch64-hdf5
+        - CIBW_ARCHS: aarch64
 
-before_install:
-  # - export PATH=/usr/lib/ccache:$PATH
-  - ccache -s
+  before_install:
+    # - export PATH=/usr/lib/ccache:$PATH
+    - ccache -s
```

Comment:
This doesn't look quite right - the indentation puts it under `matrix:` . I think `before_install` should be at the same level as `install`.